commit 98536fedd0d3ed89eecb9993cf39dc235d58558f
Author: Krzysztof Opasiak <kopasiak90@gmail.com>
Date:   Thu Jan 19 18:55:27 2017 +0100

    usb: gadget: f_hid: fix: Free out requests
    
    commit 20d2ca955bd09639c7b01db5761d157c297aea0a upstream.
    
    Requests for out endpoint are allocated in bind() function
    but never released.
    
    This commit ensures that all pending requests are released
    when we disable out endpoint.
    
    Fixes: 99c515005857 ("usb: gadget: hidg: register OUT INT endpoint for SET_REPORT")
    Tested-by: David Lechner <david@lechnology.com>
    Signed-off-by: Krzysztof Opasiak <k.opasiak@samsung.com>
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1313_98536fedd0d3ed89eecb9993cf39dc235d58558f.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1313_98536fedd0d3ed89eecb9993cf39dc235d58558f.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
